R/pkg/inst/tests/testthat/test_mllib_clustering.R

```diff
@@ -99,7 +99,8 @@ test_that("spark.kmeans", {
 
   take(training, 1)
 
-  model <- spark.kmeans(data = training, ~ ., k = 2, maxIter = 10, initMode = "random")
+  model <- spark.kmeans(data = training, ~ ., k = 2, maxIter = 10, initMode = "random", seed = 1,
+                        initSteps = 3, tol = 1E-5)
```

Comment:
I tested existing unit tests in KmeansSuite.scala in both ML and MLLIB. They are all insensitive to `seed` and `tol`.